tini/src/seccomp_fd_notify.c

```diff
@@ -400,30 +385,31 @@ void maybe_setup_seccomp_notifer() {
 			return;
 		}
 
-		/* 
-		   We need Tini to be ready to intercept and pass through the first 
-		   sendmsg() system call before sending the notify fd. 
-		   So, we wait until the polling has been set up.
-		*/
-		pthread_create(&thread_id, NULL, &catch_send_fd, (void *)&notify_fd);
-		pthread_mutex_lock(&wait_to_send);
-		while(send_notify_fd == 0) {
-			pthread_cond_wait(&ready_to_send, &wait_to_send);
+		char *is_handle_net = getenv("TITUS_SECCOMP_AGENT_HANDLE_NET_SYSCALLS");
```

Comment:
Maybe longer term we can have an enum for the types, just kinda awkward when we only have 2.
*then* it can be a param to `install_notify_filter` and it can know which type

Reply:
agreed.